txt2tags.py

```diff
@@ -1814,7 +1814,7 @@ def Readfile(file_path):
         try:
             with open(file_path) as f:
                 data = f.readlines()
-        except Exception as exception:
+        except IOError as exception:
```

Comment:
IOError should be OSError nowadays: https://stackoverflow.com/questions/29347790/difference-between-ioerror-and-oserror

Reply:
Oh, nice. Just force-pushed an updated version with `OSError`. Also changed in `Savefile()` for consistency.

Reply:
Mmmmm, in Python 2 it's still `IOError` for common failures such as "file not found":

```diff
-txt2tags: Error: Cannot read file: XXX.inc
-[Errno 2] No such file or directory: 'XXX.inc'
+Sorry! Txt2tags aborted by an unknown error.
+Please send the following Error Traceback to the author (jendrikseipp@gmail.com):
+
+Traceback (most recent call last):
+  File "/home/travis/build/txt2tags/txt2tags/txt2tags.py", line 4997, in <module>
+    exec_command_line()
+  File "/home/travis/build/txt2tags/txt2tags/txt2tags.py", line 4934, in exec_command_line
+    CMDLINE_RAW = CommandLine().get_raw_config(cmdline_data, relative=True)
+  File "/home/travis/build/txt2tags/txt2tags/txt2tags.py", line 1997, in get_raw_config
+    ret.extend(ConfigLines().include_config_file(value))
+  File "/home/travis/build/txt2tags/txt2tags/txt2tags.py", line 2488, in include_config_file
+    lines = self.read_config_file(file_)
+  File "/home/travis/build/txt2tags/txt2tags/txt2tags.py", line 2470, in read_config_file
+    lines = Readfile(filename)
+  File "/home/travis/build/txt2tags/txt2tags/txt2tags.py", line 1815, in Readfile
+    with open(file_path) as f:
+IOError: [Errno 2] No such file or directory: 'XXX.inc'
+
```

Supporting both Python 2 ans 3 is such a burden...

I've changed it to `except (OSError, IOError) as exception:`. If you prefer to revert back to `IOError` only, just tell me.

Reply:
Well, flake8 did not like my new version, so I've reverted to be original solo `IOError` :)

```
style run-test: commands[1] | flake8 --max-line-length=90 --extend-ignore E402 --exclude .tox .
./txt2tags.py:1817:9: B014 Redundant exception types in `except (OSError, IOError) as exception:`.  Write `except () as exception:`, which catches exactly the same exceptions.
```